Formula/a/asymptote.rb

```diff
@@ -55,6 +55,9 @@ def install
     system "make", "asy-keywords.el"
     system "make", "install-asy"
 
+    # https://github.com/vectorgraphics/asymptote/issues/497
+    chmod "+x", opt_bin/"xasy"
```

Comment:
```suggestion
    chmod "+x", pkgshare/"GUI/xasy.py"
```
Let's try making the symlink target executable instead.

Reply:
Never thought this way, and thank you again!